{
  expectedException.expect(IllegalArgumentException.class);
  expectedException.expectMessage("Error can be provided only when status is FAILED");
  underTest.remove(mock(CeTask.class),CeActivityDto.Status.CANCELED,null,new RuntimeException("Some error"));
}
